src/main/java/de/tum/in/www1/artemis/service/ParticipationService.java

```diff
@@ -261,7 +261,7 @@ private StudentParticipation startProgrammingExercise(ProgrammingExercise exerci
         // (when the web hook was already initialized, see below)
         participation = performEmptyCommit(participation);
         // Note: we configure the repository webhook last, so that the potential empty commit does not trigger a new programming submission (see empty-commit-necessary)
-        // Step 1c) configure the web hook of the student repository
+        // Step 3) configure the web hook of the student repository
```

Comment:
This was intentionally steps 1c) because it belongs to all steps 1 which related to the repository configuration, but I guess we can also live with Step 3